scripts/r.in.srtm/r.in.srtm.py

```diff
@@ -270,7 +264,7 @@ def main():
     outf.close()
 
     try:
-        grass.run_command('r.in.gdal', input=bilfile, out=tileout)
+        grass.run_command('r.import', input=bilfile, out=tileout, resample='bilinear')
```

Comment:
Better use bilinear_f (bilinear interpolation with fallback), otherwise you loose cells at the borders or next to NULL cells.